docs/dev/howtos/locally-published-components-in-fenix.md

```diff
@@ -0,0 +1,101 @@
+# Using locally-published Glean in Fenix
+
+> Note: This is a bit tedious, and you might like to try the substitution-based > approach documented in
+> [Working on unreleased Glean code in android-components](./development-with-android-components.md).
+> That approach is still fairly new, and the local-publishing approach in this document is necessary if it fails.
+
+> Note: This is Fenix-specific only in that some links on the page go to the `mozilla-mobile/fenix` repository,
+> however these steps should work for e.g. `reference-browser`, as well.
+> (Same goes for lockwise, or any other consumer of Glean, but they may use a different structure -- lockwise has no Dependencies.kt, for example)
+
+## Preparation
+
+Clone the Glean SDK, android-components and Fenix repositories:
+
+```sh
+git clone https://github.com/mozilla/glean
+git clone https://github.com/mozilla-mobile/android-components
+git clone https://github.com/mozilla-mobile/fenix/
+```
+
+## Local publishing
+
+
+1. Inside the `glean` repository root:
+    1. In [`.buildconfig.yml`](glean-yaml), change
+       `libraryVersion` to end in `-TESTING$N` <sup><a href="#note1">1</a></sup>,
```

Comment:
I think mdbook supports the [markdown footnote syntax](https://www.markdownguide.org/extended-syntax/#footnotes).  (It's used elsewhere in our docs).

Reply:
True, let me try that